src/components/AccountAvatar/AccountAvatar.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import AccountAvatar from ".";
+
+describe("AccountAvatar component", () => {
+  test("should render account avatar and check for dropdown menu button", () => {
+    render(<AccountAvatar />);
+    const accountAvatar = screen.getByLabelText("Account settings");
+    expect(accountAvatar).toBeInTheDocument();
+  });
+
+  test("should render account avatar and show dropdown menu on button click", () => {
+    render(<AccountAvatar />);
+    const button = screen.getByRole("button");
+
+    fireEvent.click(button);
```

Comment:
One of the suggestion from the creator of react testing library is to use `@testing-library/user-event` over `fireEvent` where possible. `@testing-library/user-event` is a package that's built on top of `fireEvent`, but it provides several methods that resemble the user interactions more closely. Please check [here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event) for more information.